enterprise_access/settings/base.py

```diff
@@ -310,10 +312,12 @@ def root(*path_fragments):
 SYSTEM_TO_FEATURE_ROLE_MAPPING = {
     SYSTEM_ENTERPRISE_OPERATOR_ROLE: [
         SUBSIDY_ACCESS_POLICY_OPERATOR_ROLE,
+        CONTENT_ASSIGNMENTS_OPERATOR_ROLE,
         REQUESTS_ADMIN_ROLE,
     ],
     SYSTEM_ENTERPRISE_ADMIN_ROLE: [
         SUBSIDY_ACCESS_POLICY_LEARNER_ROLE,
+        CONTENT_ASSIGNMENTS_ADMIN_ROLE,
         REQUESTS_ADMIN_ROLE,
     ],
     SYSTEM_ENTERPRISE_LEARNER_ROLE: [
```

Comment:
Just to be clear, learners will not get any access to ANY content assignment APIs!